src/cascadia/TerminalApp/Tab.h

```diff
@@ -5,11 +5,15 @@
 #include <winrt/Microsoft.UI.Xaml.Controls.h>
 #include "Pane.h"
 
-class Tab
+// TODO: make Tab a WinRT type

```

Comment:
We typically like to avoid bare todos in the code like this, without an issue linked to them. Either we should file a follow-up task (and link it here with `// TODO:microsoft/terminal#<issue_number>: Make Tab a WinRT Type`, or just pull the todo entirely. Frankly I don't think this one needs to be in the code, but go with what you feel 😄

Reply:
Sorry about that, I'll remove it. I wasn't aware of the todo guideline. Let's stick with repo's format for todos.

Reply:
Sorry about that, I'll remove this. Best to stick with the standard todo format.

Reply:
Thanks!